packages/styles/scss/components/callout-quote/_callout-quote.scss

```
@@ -53,4 +35,70 @@
       outline: transparent;
     }
   }
+
+  :host(#{$c4d-prefix}-callout-quote[color-scheme='inverse']) {
+    @extend :host(#{$c4d-prefix}-callout[color-scheme='inverse']);
+    @extend :host(#{$c4d-prefix}-callout-text[color-scheme='inverse']);
+  }
+
+  :host(#{$c4d-prefix}-callout-quote[color-scheme='layer']) {
+    @extend :host(#{$c4d-prefix}-callout[color-scheme='layer']);
+    @extend :host(#{$c4d-prefix}-callout-text[color-scheme='layer']);
+  }
+
+  :host(#{$c4d-prefix}-callout-quote[color-scheme='purple']) {
+    @extend :host(#{$c4d-prefix}-callout[color-scheme='purple']);
+    @extend :host(#{$c4d-prefix}-callout-text[color-scheme='purple']);
+  }
+  :host(#{$c4d-prefix}-callout-quote[color-scheme='cyan']) {
+    @extend :host(#{$c4d-prefix}-callout[color-scheme='cyan']);
+    @extend :host(#{$c4d-prefix}-callout-text[color-scheme='cyan']);
+  }
+
+  :host(#{$c4d-prefix}-callout-link-with-icon) {
+    @extend .#{$prefix}--link-with-icon;
+
+    display: inline-block;
+    a.#{$prefix}--link-with-icon.#{$prefix}--link.#{$prefix}--link-with-icon__icon-right {
+      color: $link-primary;
+      outline-color: $focus-inverse;
+
+      &:hover {
+        // @carbon/styles has no hover color for $inverse-link.
+        // Solution is to use the same token and add underline
+        color: $link-primary;
+        text-decoration: underline;
+      }
+
+      &:active {
+        color: $link-primary;
+      }
+    }
+  }
+
+  :host(#{$c4d-prefix}-callout-link-with-icon[color-scheme='inverse']) {
+    a.#{$prefix}--link-with-icon.#{$prefix}--link.#{$prefix}--link-with-icon__icon-right {
+      color: $link-inverse;
+
+      &:hover {
+        // @carbon/styles has no hover color for $inverse-link.
+        // Solution is to use the same token and add underline
+        color: $link-inverse;
```

```suggestion
        color: $link-inverse;
        text-decoration: underline;
```
Should we also add the underline here?

---

i actually removed underline from both... there's already styles adding the underline from other imported styles, so the underline still is visible